### ISSUE TITLE
Halted notification only says how many are needed for one iteration 

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#15453]
![image](https://user-images.githubusercontent.com/3536496/77977183-d666df80-72b3-11ea-8a6a-10b96b2e8384.png)

**Comment by beastyn** [comment]
First halted message was fine, I needed 22 logs and 300 labor.
After I've added 2 birch logs I had halted message only with labor
![bandicam 2020-04-03 12-41-49-817](https://user-images.githubusercontent.com/27898520/78336347-a74cb880-75a8-11ea-82c8-5eab808096a7.jpg)

**Comment by beastyn** [comment]
Or is it inteded? Like  in current moment it needs only labor to proceed.
Kind of strange. In the first message it tells you what do you need in general. And then it tells you about just current iteration,

**Comment by mirasrael** [comment]
I guess it is intended, because it shows what is blocking your order from progress. But I agree  current dual behavior is confusing. So easiest fix for me is show both current iteration and total resources. Like 
"Needs 2 Logs and 20 Labor for current iteration (22 Logs and 300 Labor in total)".

**Comment by johnkslg** [comment]
Fixed by saying everything left to add whenever it halts.

**Comment by beastyn** [comment]
Still a bit odd
1. I've created an order for 10 hewm logs
![bandicam 2020-04-09 15-09-09-664](https://user-images.githubusercontent.com/27898520/78884388-43c4fe00-7a74-11ea-884b-82150db57095.jpg)
So, its talking about first itaration and whats left. Ok
2. I've added 2 birch logs for the first iteration
![bandicam 2020-04-09 15-09-33-858](https://user-images.githubusercontent.com/27898520/78884595-9d2d2d00-7a74-11ea-9f8a-4b1ee6a3c443.jpg)
Now I have 10 Hewn logs in lefrowers... Why?
